code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2409,40 +2409,40 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	
 /datum/reagent/consumable/ethanol/sea_breeze
 	name = "Sea Breeze"
-	description = "Light and refreshing with a mint and cocoa hit- like mint choc chip ice cream you can drink!"
+	description = "Light and refreshing with a hint of mint and cocoa. Sweet, like a smoothie."
 	boozepwr = 15
 	color = "#CFFFE5"
 	quality = DRINK_VERYGOOD
 	taste_description = "mint choc chip"
 	glass_icon_state = "sea_breeze"
 	glass_name = "Sea Breeze"
-	glass_desc = "Minty, chocolatey, and creamy. It's like drinkable mint chocolate chip!"
+	glass_desc = "A mint-chocolate, creamy shake."
 
 /datum/reagent/consumable/ethanol/sea_breeze/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
 	M.apply_status_effect(/datum/status_effect/throat_soothed)
 	..()
 
 /datum/reagent/consumable/ethanol/white_tiziran
 	name = "White Tiziran"
-	description = "A mix of vodka and kortara. The Lizard imbibes."
+	description = "A mix of vodka and kortara, often utilized during vuulek celebrations."
 	boozepwr = 65
 	color = "#A68340"
 	quality = DRINK_GOOD
 	taste_description = "strikes and gutters"
 	glass_icon_state = "white_tiziran"
 	glass_name = "White Tiziran"
-	glass_desc = "I had a rough night and I hate the fucking humans, man."
+	glass_desc = "A sweet mint vodka with a hint of cocoa."
 
 /datum/reagent/consumable/ethanol/drunken_espatier
 	name = "Drunken Espatier"
```

Comment:
Also lore although i dont think this would be as easy to handle, unless an espatier isnt lore